biogeophys/FatesHydroWTFMod.F90

```diff
@@ -479,20 +514,20 @@ function psi_from_th_vg(this,th) result(psi)
     !
     ! *also modified to accomodate linear pressure regime for super-saturation
     ! -----------------------------------------------------------------------------------
-
+    thx = max(th, this%th_res+1e-6)
```

Comment:
Same comment as above.  Could we move these changes to something that is coordinated with you and @JunyanDing.  Perhaps submit a PR with these changes to the branch she is using for #737 ?

Reply:
This can be merged with Junyan's changes.